pydocx/__main__.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+import sys
+import logging
+
+from pydocx.parsers.Docx2Html import Docx2Html
+from pydocx.parsers.Docx2Markdown import Docx2Markdown
+
+
+def convert(parser_type, docx_path, output_path):
+    if parser_type == '--html':
+        output = Docx2Html(docx_path).parsed
+    elif parser_type == '--markdown':
```

Comment:
In one of your commit messages you said Markdown support wasn't great, and skipped that test. Do you still want to keep the flag active here?